packages/account/src/Helpers/utils.tsx

```diff
@@ -46,7 +47,13 @@ export const shouldShowIdentityInformation = ({
 }: TIDVSupportCheck) => {
     const country = residence_list.find(item => item.value === citizen);
     const maltainvest = real_account_signup_target === 'maltainvest';
-    const should_skip_idv = account_status?.status?.some((status: string) => status === 'skip_idv'); //status added by BE when idv should be skipped for the user
+    const identity = account_status?.authentication?.identity;
+    const is_onfido_verified = identity?.services?.onfido?.status === AUTH_STATUS_CODES.verified;
+    const is_manual_verified = identity?.services?.manual?.status === AUTH_STATUS_CODES.verified;
+    const is_identity_verified = identity?.status === AUTH_STATUS_CODES.verified;
+    const should_skip_idv =
+        ((is_onfido_verified || is_manual_verified) && is_identity_verified) ||
```

Comment:
why do we need this extra condition check for manual and idv? How is it different from overall POI status check?

Reply:
Basically for DIEL clients, if MF accounts is verified through either onfido or manual, we should not ask for IDV again for CR account creation and I made it explicitly to onfido or manual